contributors/devel/sig-instrumentation/logging.md

```diff
@@ -4,14 +4,14 @@ The following conventions for the klog levels to use.
 [klog](http://godoc.org/github.com/kubernetes/klog) is globally preferred to
 [log](http://golang.org/pkg/log/) for better runtime control.
 
-Shared libraries, such as `client-go`, should not use `klog.Errorf()` and `klog.Warningf()`,
+Shared libraries, such as `client-go`, should not use `klog.ErrorS()` and `klog.InfoS()`,
 but just return `error`, because client libraries may be used in CLI UIs that wish to control output.
```

Comment:
```suggestion
but just return `error`, because client libraries may be used in CLI UIs that wish to control output.

Please see the [Structured Logging Guide](contributors/devel/sig-instrumentation/migration-to-structured-logging.md#structured-logging-in-kubernetes) for more information on how to set keys and values in structured logs.
```